backend/src/game/Player.ts

```diff
@@ -15,4 +15,8 @@ export class Player extends User {
   public getCards(): Card[] {
     return this.cards;
   }
+
+  public foldCards(): void {
```

Comment:
Yeah, I think since it was ultimately the Dealer action I tried to make it consistent with the dealCards function. But we can go multiple ways with it.